### ISSUE TITLE
Locks up after copying to clipboard 

### ISSUE THREAD

**Issue by TinaWilde** [Eve-PySpy/PySpy#24]
After selecting the character names from the local chat and pressing ctrl C PySpy says resolving character names to IDs and goes no further.

https://prnt.sc/xhTcmlwM6iT8

Not sure what has changed, worked fine the night before but does not seem to want to now.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**System Environment:**
 - OS: [e.g. macOS, Windows]
 - OS Version: [e.g. High Sierra, 10]
 - PySpy Version [e.g. 0.1]

**Additional context**

**Comment by a56444c** [comment]
I experienced this the first time I tried this tool about a week ago. I guess it's truly dead then.

**Comment by bkmvlgit** [comment]
Yeah i have the same issue, running in Windows 10, Dual Xeon 128 gig ram, 250mbit fibre connection, PySpy version is v0.5.2.
Seems it stopped working after the last EVE patch, was fine before that and i use it on a daily basis.

**Comment by harveymw** [comment]
I too have seen this and it has been working.

**Comment by harveymw** [comment]
I pulled it down. going to see if I can figure out why

**Comment by harveymw** [comment]
Well that was a quick excursion. Its failing because it is attempting to contact its backend db.
from the src :
Query PySpy's proprietary kill database for the character ids provided as a list or tuple of integers. Returns a JSON containing one line per character id.

This db was hosted here: http://pyspy.pythonanywhere.com
and that page is gone. so the owner has either ran out of funds to host it or is not interested in doing it anymore.

**Comment by harveymw** [comment]
from looking at the project it is abandoned. There is no project members.

**Comment by bkmvlgit** [comment]
Well that is not great, gotten so used to this tool guess pirates little helper might still work, copy and pasting info manually from local to zkill is a fair bit of tedium, guess you could set up your own database but not sure what he was using i am mostly a DB2 guy myself.

**Comment by harveymw** [comment]
Well he has another project that is the pyspy backend. its what was hosted. would appear that he uses sqlite and mongo for kill mail storing. This can all be setup local. I will likely attempt to do that.

**Comment by leonahess** [comment]
Hey y'all,

yeah this project is unmaintained. Both me and the other maintainer don't have the time to work on it anymore. I was providing daily Database updates until now but will also stop doing that now.

**Comment by bkmvlgit** [comment]
Well thanks for keeping it going i noticed Pirates little helper is also not updated anymore, not sure how hard it is to set this up on your own but not a python programmer to begin with.

**Comment by FreeSide88** [comment]
I did some work and managed to bring pyspy to life.

From what I did:
- redesigned backend(moved mysql to postgresql, and some improvements with sqlite, mongodb; improved data collector)
- collected the base over the past three months, nothing prevents expanding
- reworked some of the pyspy requests from get to post

anyone interested in pyspy?
If there is feedback, I'm willing to share the code and support it to the best of my ability.

**Comment by harveymw** [comment]
I liked using it. If you could share code that would be cool. I too took a look at how it was put together with the notion of just running local.

**Comment by FreeSide88** [comment]
I made a similar server http://pyspy.pythonanywhere.com
Wondering how best to share the code? the pull request is unlikely to be accepted or to make a fork?

**Comment by bkmvlgit** [comment]
Would love to use it again if possible.

**Comment by harveymw** [comment]
I think it best to make a fork.

**Comment by FreeSide88** [comment]
however, at the moment I'm running the client through python3, not through exe. It may not be convenient for many. I will work through the issue with launching through the .exe file, as it was before.

**Comment by bkmvlgit** [comment]
Well a bat file would be a quick way to do that

**Comment by harveymw** [comment]
or pyinstaller

**Comment by FreeSide88** [comment]
I successfully forked, uploaded the code and made a working exe. you can see it in my profile. and I also expanded the database for the whole year.

**Comment by bkmvlgit** [comment]
Thanks, besides AVG going crazy with the file it seems to run alright, and the lookup seems to work to.
Did you update the about part with link to your version?

**Comment by FreeSide88** [comment]
Yes, iam updated about information.

**Comment by bkmvlgit** [comment]
Well the about window points to the original download link, which i guess is fine.

**Comment by FreeSide88** [comment]
i fix it

**Comment by jhmartin** [comment]
I've forked and resurrected the backend  and frontend, updated frontend at https://github.com/jhmartin/PySpy/releases/tag/v0.5.6
